CONTRIBUTING.md

```diff
@@ -375,19 +382,6 @@ git rebase master
 
 Finally, open a [new Pull Request](https://github.com/webex/webex-js-sdk/compare) with your changes. Be sure to mention the issues this request addresses in the body of the request. Once your request is opened, a developer will review, comment, and, when approved, merge your changes!
 
-### Pull Request Checklist
```

Comment:
Is there a reason for removing the checklist?

Reply:
Guess it wasn't rebased correctly